pkg/utils/common.go

```diff
@@ -455,6 +456,25 @@ func GetChainIDs(endpoint string, chainName string) (string, string, error) {
 	return "", "", fmt.Errorf("%s not found on primary network blockchains", chainName)
 }
 
+func GetNodeID(endpoint string) (
+	string, // nodeID
+	string, // public key
+	string, // PoP
```

Comment:
should we just return a nodeID object here, instead of three strings?

Reply:
I prefer this reformatting to be done on another PR

Reply:
https://github.com/ava-labs/avalanche-cli/issues/2702